wof/examples/flask/odm2/timeseries/sqlalch_odm2_models.py

```diff
@@ -152,9 +161,10 @@ def __init__(self,aff_obj):
         self.Phone = aff_obj.PrimaryPhone
         self.Email = aff_obj.PrimaryEmail
         self.Address = aff_obj.PrimaryAddress
-        #self.City = 'San Diego'
-        #self.State = 'CA'
-        #self.ZipCode = '92122'
+        # self.City = 'San Diego'
+        # self.State = 'CA'
+        # self.ZipCode = '92122'
+
```

Comment:
Sigh. I must not be in a github review mode :(  For some reason I thought you were deleting the commented out lines. Ignore my comments, then.